test/unittests/libdevcrypto/LibSnark.cpp

```diff
@@ -338,6 +334,155 @@ BOOST_AUTO_TEST_CASE(pairingNullInput)
 	BOOST_CHECK(!r.first);
 }
 
+BOOST_AUTO_TEST_CASE(generateRandomPoints)
```

Comment:
I don't get why the tests for EC stuff are being added in this PR? Some anomaly during rebase?